test/Groups/matrixgroups.jl

```diff
@@ -266,6 +266,12 @@ end
    @test K==matrix_group(x.elm, (x^2).elm, y.elm)
    @test K==matrix_group([x.elm, (x^2).elm, y.elm])
 
+   n = nrows(x)
+   matgens = typeof(x)[]   # empty list of generators
+   G = MatrixGroup(n, F)
+   G.gens = [MatrixGroupElem(G, mat) for mat in matgens]
```

Comment:
I think this needs a prefix on the [] to help with the type when the array is empty.

Reply:
Should hopefully be `elem_type(G)[...]`.

Reply:
The tests with Julia 1.6 pass, and I had checked the change only with this version.
So I will install Julia 1.7 ...